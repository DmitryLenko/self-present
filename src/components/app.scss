@import '~reset-css/sass/reset';

@import '../styles/mixins';

.mainWrapper {
  padding: 20px;
  box-sizing: content-box;
  .stats {
    @include base;
    .statsItem {
      padding: 5px 0;
    }
  };
  .tasksListWrapper {
    @include base;
    .taskItemWrapper {
      display: flex;
      padding: 5px 0;
    }
  };
}
