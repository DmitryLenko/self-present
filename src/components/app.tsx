import React from 'react'

import './app.scss'

export const HelloWorld = () => (
         <div>
           <div className="test">Hello world</div>
         </div>
       )
