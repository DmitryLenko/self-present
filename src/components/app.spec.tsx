import React from "react";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";

import { HelloWorld } from "./app";

describe('app', () => {
  it('base', () => {
    expect(toJson(mount(<HelloWorld />))).toMatchSnapshot();
  });
});
