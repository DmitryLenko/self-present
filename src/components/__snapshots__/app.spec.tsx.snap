// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app base 1`] = `
<HelloWorld>
  <div>
    <div
      className="test"
    >
      Hello world
    </div>
  </div>
</HelloWorld>
`;
